code/lib/eslint-plugin/vitest.config.ts

```diff
@@ -1,15 +1,13 @@
-import { defineConfig } from 'vitest/config';
+import { defineConfig, mergeConfig } from 'vitest/config';
 
-export default defineConfig({
-  test: {
-    name: 'eslint-plugin-storybook',
-    include: ['**/*.test.ts'],
-    environment: 'node',
-    globals: true,
-    coverage: {
-      provider: 'v8',
-      reporter: ['text', 'lcov'],
-      exclude: ['dist/**', 'tools/**'],
+import { vitestCommonConfig } from '../../vitest.shared';
+
+export default mergeConfig(
+  vitestCommonConfig,
+  defineConfig({
+    test: {
+      // This is needed because @typescript-eslint/rule-tester API requires it under the hood
```

Comment:
globals means the `test`, `it` etc, will be available globally, right?
I honestly wish we didn't have to do it, is there some long term replacement of `@typescript-eslint/rule-tester` or an upgrade of the package we could switch to using?

Can we open an issue with them requesting this?